src/components/popovers/popover.stories.mdx

```diff
@@ -54,6 +54,25 @@ Base popover component. Bring your own trigger and popover content.
   </Popover>
 </Story>
 
+<Preview isExpanded>
+  <Popover
+    position='right'
+    alignment='start'
+    distance={4}
+    interactionMode='click'
+    transitionName="GrowFast"
+    trigger={<div className="margin--bottom"><Button label="Click Me" /></div>}
+  >
+    <div
```

Comment:
we can simplify this example by removing the styled div here

Reply:
My only concern is that all the simplified stories work as a live example. If we put nothing in here, Popover will seem to not work. If we put just text in here, it'll get pushed all the way to the right side of the screen.